plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizard.java

```diff
@@ -20,12 +39,18 @@
 
 public class StandardProjectWizard extends Wizard implements INewWizard {
 
-  private AppEngineStandardWizardPage page;
+  private WizardPage page;
   private AppEngineStandardProjectConfig config = new AppEngineStandardProjectConfig();
+  private boolean hasAppEngineComponent;
 
   public StandardProjectWizard() {
     this.setWindowTitle("New App Engine Standard Project");
-    page = new AppEngineStandardWizardPage();
+    hasAppEngineComponent = doesAppEngineJavaComponentExist();
+    if (hasAppEngineComponent) {
```

Comment:
We thought it would be simpler to have the user cancel out of the wizard and restart, rather than adding a button whose only purpose was to check for the component.

If there were a way the wizard could notice the component was installed and enable the next button, it would certainly be better to check in a single page.